### DISCUSSION TITLE
書き込みテストスレ

### DISCUSSION THREAD

**Discussion by az1979** [gameusers/node#1] [General / 一般]
## 👋 Welcome!
  なんでも気軽に書き込んでください。
  * 疑問に思っていることを質問する。
  * アイデアを共有する。
  * 他のコミュニティメンバーと交流する。
  * 他者を歓迎し、オープンマインドであること。これは、私たちが一緒に作るコミュニティであることを忘れないでください。
  一緒に作っていくコミュニティであることを忘れないでください💪

上記は Discussions のテンプレートを翻訳しました。

**Comment by AmazyDayzee** [comment]
hello :)

**Reply by az1979** [reply]
:laughing:
